Appveyor build is broken

Currently, `clang` for Appveyor is `6.0.0` and the error message shows that we need a newer version

```
TARGET = Some("x86_64-pc-windows-msvc")
running: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\VC\\Tools\\MSVC\\14.16.27023\\bin\\HostX64\\x64\\lib.exe" "/OUT:C:\\projects\\servo\\target\\debug\\build\\mozjs_sys-0c9b40496f94e80f\\out\\libjsglue.a" "/nologo" "C:\\projects\\servo\\target\\debug\\build\\mozjs_sys-0c9b40496f94e80f\\out\\src\\jsglue.o"
exit code: 0
TARGET = Some("x86_64-pc-windows-msvc")
HOST = Some("x86_64-pc-windows-msvc")
TARGET = Some("x86_64-pc-windows-msvc")
TARGET = Some("x86_64-pc-windows-msvc")
HOST = Some("x86_64-pc-windows-msvc")
CXX_x86_64-pc-windows-msvc = None
CXX_x86_64_pc_windows_msvc = None
HOST_CXX = None
CXX = None
cargo:rustc-link-lib=static=jsglue
cargo:rustc-link-search=native=C:\projects\servo\target\debug\build\mozjs_sys-0c9b40496f94e80f\out
TARGET = Some("x86_64-pc-windows-msvc")
HOST = Some("x86_64-pc-windows-msvc")
CXXSTDLIB_x86_64-pc-windows-msvc = None
CXXSTDLIB_x86_64_pc_windows_msvc = None
HOST_CXXSTDLIB = None
CXXSTDLIB = None
TARGET = Some("x86_64-pc-windows-msvc")
Generting bindings ["./src/jsglue.hpp", "--rust-target", "1.25", "--rustified-enum", ".*", "--no-derive-default", "--enable-cxx-namespaces", "--generate", "functions,types,vars", "--ignore-methods", "--", "-I", "C:\\projects\\servo\\target\\debug\\build\\mozjs_sys-0c9b40496f94e80f\\out\\dist/include", "-I", "C:\\projects\\servo\\target\\debug\\build\\mozjs_sys-0c9b40496f94e80f\\out\\js/src", "-x", "c++", "-fms-compatibility", "-DRUST_BINDGEN", "-DSTATIC_JS_API", "-std=c++14", "-DWIN32", "-D_CRT_USE_BUILTIN_OFFSETOF", "-include", "C:\\projects\\servo\\target\\debug\\build\\mozjs_sys-0c9b40496f94e80f\\out\\js/src/js-confdefs.h", "--rustfmt-configuration-file", "rustfmt.toml"] clang version 6.0.0 (tags/RELEASE_600/final).
--- stderr
which: python2.7: unknown command
Creating config.status
Reticulating splines...
Finished reading 38 moz.build files in 0.17s
Read 0 gyp files in parallel contributing 0.00s to total wall time
Processed into 182 build config descriptors in 0.66s
RecursiveMake backend executed in 0.21s
  126 total backend files; 126 created; 0 updated; 0 unchanged; 0 deleted; 6 -> 31 Makefile
Total wall time: 1.13s; CPU time: 1.13s; Efficiency: 100%; Untracked: 0.08s
Microsoft (R) C/C++ Optimizing Compiler Version 19.16.27025.1 for x64
Copyright (C) Microsoft Corporation.  All rights reserved.
self-hosting-cpp-input.cpp
   Creating library nspr4.lib and object nspr4.exp
   Creating library plds4.lib and object plds4.exp
   Creating library plc4.lib and object plc4.exp
   Creating library mozjs-60.lib and object mozjs-60.exp
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.16.27023\include\yvals_core.h:294:5: error: STL1000: Unexpected compiler version, expected Clang 7 or newer.
C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.16.27023\include\yvals_core.h:294:5: error: STL1000: Unexpected compiler version, expected Clang 7 or newer., err: true
thread 'main' panicked at 'Should generate JSAPI bindings OK: ()', libcore\result.rs:1009:5
stack backtrace:
   0: <std::sync::barrier::BarrierWaitResult as core::fmt::Debug>::fmt
   1: <std::path::Iter<'a> as core::convert::AsRef<std::path::Path>>::as_ref
   2: std::panicking::take_hook
   3: std::panicking::take_hook
   4: std::panicking::rust_panic_with_hook
   5: std::panicking::begin_panic_fmt
   6: rust_begin_unwind
   7: core::panicking::panic_fmt
   8: core::result::unwrap_failed<()>
             at \libcore\macros.rs:26
   9: core::result::Result<bindgen::Bindings, ()>::expect<bindgen::Bindings,()>
             at \libcore\result.rs:835
  10: build_script_build::build_jsapi_bindings
             at .\build.rs:251
  11: build_script_build::main
             at .\build.rs:22
  12: std::rt::lang_start::{{closure}}<()>
             at \libstd\rt.rs:74
  13: std::panicking::update_panic_count
  14: _rust_maybe_catch_panic
  15: std::rt::lang_start_internal
  16: std::rt::lang_start<()>
             at \libstd\rt.rs:74
  17: main
  18: invoke_main
             at d:\agent\_work\3\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:78
  19: invoke_main
             at d:\agent\_work\3\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:78
  20: BaseThreadInitThunk
  21: RtlUserThreadStart
```

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,5 +1,5 @@
 version: 1.0.{build}
-image: Visual Studio 2017
+image: Previous Visual Studio 2017
 
 environment:
   CCACHE_DIR: "%APPVEYOR_BUILD_FOLDER%\\.ccache"
@@ -17,7 +17,6 @@ environment:
     C:\\Python27\\Scripts;\
     C:\\Tools\\PsTools;\
     C:\\Tools\\NuGet3;\
-    C:\\Program Files\\LLVM\\bin;\
     C:\\Program Files\\Microsoft\\Web Platform Installer\\;\
     C:\\Program Files\\7-Zip;\
     C:\\Program Files\\Mercurial;\

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,6 +1,7 @@
 version: 1.0.{build}
 image: Previous Visual Studio 2017
 
+
 environment:
   CCACHE_DIR: "%APPVEYOR_BUILD_FOLDER%\\.ccache"
   CARGO_INCREMENTAL: 0

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,7 +1,6 @@
 version: 1.0.{build}
 image: Previous Visual Studio 2017
 
-
 environment:
   CCACHE_DIR: "%APPVEYOR_BUILD_FOLDER%\\.ccache"
   CARGO_INCREMENTAL: 0

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -17,6 +17,7 @@ environment:
     C:\\Python27\\Scripts;\
     C:\\Tools\\PsTools;\
     C:\\Tools\\NuGet3;\
+    C:\\Program Files\\LLVM\\bin;\
     C:\\Program Files\\Microsoft\\Web Platform Installer\\;\
     C:\\Program Files\\7-Zip;\
     C:\\Program Files\\Mercurial;\

